src/train.py

```diff
@@ -60,9 +61,19 @@ def train_model(arguments, train, val):
 
 
 def train_batch(data, model, optimizer, criterion, device='cpu'):
+
+
     model.to(device)
     optimizer.zero_grad()
     out = model(data)
+    # out = th.div(out,data.y)
+    # out = th.nan_to_num(out)
+    # # label = th.div(data.y,data.y)
+    # label = th.ones(data.y.shape)
+    
+    # # error = criterion(out, data.y)
+    # # loss = th.div(error, data.y)
+    # # loss = criterion(out, data.y)
```

Comment:
are these left overs? its understand to understand what they are there for without comments, lets keep the codebase clean